### ISSUE TITLE
"Pre-trained model currently not available"

### ISSUE THREAD

**Issue by ofirshifman** [achaman2/truly_shift_invariant_cnns#9]
Can you please share your trained model weights for ResNet trained on ImageNet?
I want to run it on imageNet test set, but unfortunately can't afford training by myself right now.

I assumed it's not in the repo since I saw that "Pre-trained model currently not available.",
I'm sorry If you have it somewhere in the repo and I just didn't find it, please let me know.

Thanks!
Ofir.
